docs/backends.md

```diff
@@ -512,3 +512,43 @@ data:
   current-password-again: <path:secret-id#password#version-id>
   password-old: <path:secret-id#password#old-version-id>
 ```
+
+### 1Password Connect
+##### 1Password Connect Authentication
+Refer to the [1Password Secrets Automation overview](https://support.1password.com/secrets-automation/) for 1Password Connect usage.
+
+These are the parameters for YCL:
+
+```
+AVP_TYPE: 1passwordconnect
+AVP_OP_CONNECT_TOKEN: Your 1Password Connect access token
+AVP_OP_CONNECT_HOST: The hostname of your 1Password Connect server
```

Comment:
If `OP_CONNECT_TOKEN` and `OP_CONNECT_HOST` are env variables that 1password connect will read by default, https://github.com/1Password/connect-sdk-go#environment-variables, i would rather we strip the `AVP_` and just support the native env vars like we do with Vault and others.

This would also require adding `op_connect` to this list: https://github.com/argoproj-labs/argocd-vault-plugin/blob/main/pkg/config/config.go#L44

Reply:
Sure, that's reasonable. Will push a new commit shortly